dev/DynamicDependency/M.AM.DD.PackageDependency.cpp

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+
+#include "M.AM.DD.PackageDependency.h"
+#include "Microsoft.ApplicationModel.DynamicDependency.PackageDependency.g.cpp"
+
+#include "M.AM.DD.PackageDependencyContext.h"
+
+#include <MsixDynamicDependency.h>
+#include <wil/resource.h>
+#include <wil_msixdynamicdependency.h>
+
+#include <M.AM.Converters.h>
+
+namespace winrt::Microsoft::ApplicationModel::DynamicDependency::implementation
+{
+    PackageDependency::PackageDependency(PCWSTR id)
+    {
+        m_id = winrt::to_hstring(id);
+    }
+    PackageDependency::PackageDependency(hstring const& id) :
+        m_id(id)
+    {
+    }
+    Microsoft::ApplicationModel::DynamicDependency::PackageDependency PackageDependency::GetFromId(hstring const& id)
+    {
+        return winrt::make<implementation::PackageDependency>(id);
+    }
+    Microsoft::ApplicationModel::DynamicDependency::PackageDependency PackageDependency::Create(hstring const& packageFamilyName, Windows::ApplicationModel::PackageVersion const& minVersion)
```

Comment:
This lack of newline between function-scope-close and next-line is throwing me off.  I guess at some point we'll figure out how to throw clang-format/clang-tidy at the source (@EHO-makai - something to add to our ES wishlist.)

Reply:
LOL it throws me off too. But that's how C++/WinRT generates the skeletons so I rolled with it

Sure would be nice if  C++/WinRT generated the 'nicer' layout... Feel free to [file an ask on the C++/WinRT project](https://github.com/Microsoft/cppwinrt/issues)